src/test/localization/WIP_all.test.js

```diff
@@ -0,0 +1,137 @@
+import _ from 'lodash';
+import { localizations } from '../../localization/all';
+
+/*
+Example:
+
+const pathsArray = getKeyPathsArray(
+  {
+    a: { b: '1' },
+    c: 2,
+    d: [ 1, 2, 3, ],
+    e: {
+      f: {
+        g: { h: '' },
+        i: '',
+      },
+    },
+  }
+);
+
+console.log('pathsArray', pathsArray);
+=> [
```

Comment:
This is an interesting idea! I like the concept of 'paths'. I wonder if something more like this could be more succinct and so easier to digest:
```js
[
  a.b,
  c,
  d,
  e.f.g.h,
  e.f.i,
]
```
I assume that if items were missing, those items would be shown in a list of their own, so something were missing part way through an object chain (for example, g.h didn't exist), it could be shown as `e.f`.

Thoughts welcome!

Reply:
If this is the suggested output format, that is.

Reply:
Think of this as the EN localization object.

```
{
    a: { b: '1' },
    c: 2,
    d: [ 1, 2, 3, ],
    e: {
      f: {
        g: { h: '' },
        i: '',
      },
    },
  }
```
That fed into the function generates this:
```
[
 [ 'a' ],
  [ 'a', 'b' ],
  [ 'c' ],
  [ 'd' ],
  [ 'e' ],
  [ 'e', 'f' ],
  [ 'e', 'f', 'g' ],
  [ 'e', 'f', 'g', 'h' ],
  [ 'e', 'f', 'i' ]
]
```
I can then easily loop through that and check it against any other localization object for missing keys and values which are the same (not translated).  I can do the reverse, feed in DE for instance, get its keys and compare that to EN for any keys that shouldn't be in DE.

The output would be something like what you see in the tests below, but more friendly.  By default, we'd probably just show failures since this gets really long and allow the user to filter.

✓ 'en' path aboutPage exists in 'de'
✓ 'en' path aboutPage.aboutPageHeader exists in 'de'
✓ aboutPage.aboutPageHeader translation should not be the same in 'de'
✕ 'en' path aboutPage.benefitsLinkText exists in 'de'
✕ 'en' path aboutPage.howToUse exists in 'de'
✓ 'en' path aboutPage.howToUseHeader exists in 'de'
✓ aboutPage.howToUseHeader translation should not be the same in 'de'
✕ 'en' path aboutPage.howToUseNote exists in 'de'
✕ 'en' path aboutPage.quantLinkText exists in 'de'
✕ 'en' path aboutPage.videoLinkText exists in 'de'
✕ 'en' path aboutPage.whatFor exists in 'de'
✓ 'en' path aboutPage.whatForHeader exists in 'de'
✓ aboutPage.whatForHeader translation should not be the same in 'de'
✕ 'en' path aboutPage.whatForImportantNote exists in 'de'
✕ 'en' path aboutPage.whoMadeThis1 exists in 'de'
✕ 'en' path aboutPage.whoMadeThis2 exists in 'de'
✕ 'en' path aboutPage.whoMadeThis3 exists in 'de'
✓ 'en' path aboutPage.whoMadeThisHeader exists in 'de'
✓ aboutPage.whoMadeThisHeader translation should not be the same in 'de'

Reply:
Ah sorry about that.  Looking back at it, I can see how you'd think that now.  Especially if you don't actually run the file via Jest to see what it was dumping out to the console.  I'll try to be more conscious of my comments.